.github/workflows/sycl-linux-precommit.yml

```diff
@@ -110,6 +110,8 @@ jobs:
             use_dev_igc: ${{ contains(needs.detect_changes.outputs.filters, 'devigccfg') }}
             extra_lit_opts: --param matrix-xmx8=True --param gpu-intel-dg2=True
             env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
+            # Run only if the PR does not have the 'ci-no-devigc' label.
+            run_condition: ${{ contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') && 'false' || 'true' }}
```

Comment:
How about "skip-run:  ${{ contains(github.event.pull_request.labels.*.name, 'ci-no-devigc')"

Reply:
Fixed in 01c5a4df09765afb0069410e2feec8fe257effc6